src/yarle.ts

```diff
@@ -31,34 +31,42 @@ export const parseStream = async (options: YarleOptions): Promise<void> => {
   let noteNumber = 0;
   let failed = 0;
   let skipped = 0;
-  
+
   const notebookName = utils.getNotebookName(options.enexSource);
 
   return new Promise((resolve, reject) => {
-    
+
     const logAndReject = (error: Error) => {
       console.log(`Could not convert ${options.enexSource}:\n${error.message}`);
       ++failed;
 
       return reject();
     };
-    if (!fs.existsSync(options.enexSource))
+    if (!fs.existsSync(options.enexSource)) {
```

Comment:
Could you please push your tslint config? then we could work using the same settings. Thank you!

Reply:
Hm, I think I'm using the one that is [in the project](https://github.com/akosbalasko/yarle/blob/master/tslint.json) already. This is the "curly" rules that says "if statements must be braced".

Reply:
Hm. okay, I don't know then, I'll recheck my vscode's tslint component